python/ray/util/scheduling_strategies.py

```diff
@@ -72,9 +72,87 @@ def __init__(
         self._fail_on_unavailable = _fail_on_unavailable
 
 
+def validate_values(values):
+    for value in values:
+        if not isinstance(value, str):
+            raise ValueError("The value of lable must be string.")
+
+
+class In:
+    def __init__(self, values: List[str]):
```

Comment:
Should be `def __init__(self, *values):` so that we can do `In("a", "b")`

Reply:
- Yes, I made a mistake. Let me fix it."